instructions-for-devs.md

```diff
@@ -142,12 +144,15 @@ host = "localhost"
 
 [p2p]
 listen = true
-port = 8885
-#make sure this port is reachable from the internet
+port = 8885 # Make sure this port is reachable from the internet
```

Comment:
`# Make sure this port is reachable from the internet` Maybe we can phrase this differently and refer to firewalls and does that port have to be reachable when you just want to have outbound connections?

Reply:
Another comment about this subject:
The listen port may not be the port which is open on the firewall. for example, xud may listen on 8885 but the firewall may open 8888 and have port forwarding role from 8888 to 8885 on the xud machine. This is a very common situation (just consider 2 servers behind the FW running xud).

As such we must have a listen port and an external port which we publish to the peers.

Reply:
@offerm I think this is already handled in this PR by `p2p.addresses`. There you can specify the port that is sent in the handshake for each address.

Reply:
># A string array of reachable socket addresses for this node, port defaults to p2p.port if unspecified
addresses = [ "4.8.15.16:8885" ]

Well done @sangaman , that's exactly how it should be! This will allow even people in heavily censored countries to propagate `p2p.addresses` with port 443

Reply:
@kilrau The port has to be between `1024` and `65535`. But there are still plenty options to choose from. 

And when you are in a heavily censored country changing the port won't help you much. Maybe we can provide a tutorial for setting up XUD with Tor in the future.